src/allmydata/storage/server.py

```diff
@@ -49,6 +49,9 @@
 NUM_RE=re.compile("^[0-9]+$")
 
 
+# Number of seconds to add to expiration time on lease renewal:
+DEFAULT_RENEWAL_TIME = 31 * 24 * 60 * 60
```

Comment:
I'm not sure it's so much a "default" as "the only time that is possible".  Although I would love for lease time to be a parameter eventually...